docs/fedora-releases-guide/what-to-know.md

```diff
@@ -0,0 +1,112 @@
+---
+title: What maintainers should know before adopting Packit for downstream release automation 
+sidebar_position: 5
+---
+# What maintainers should know before adopting Packit for downstream release automation 
+
+## Responsibilities
+
+Packit is designed to **assist** maintainers, not **replace** them.
+
+Maintainers retain key responsibilities:
+- reviewing and merging Packit-created pull requests
+- maintaining package quality
+- ensuring updates don't break dependant packages (Packit does not have yet a reverse dependencies check)
```

Comment:
```suggestion
- ensuring updates don't break dependant packages
```

I wouldn't mention that, even if Packit already did rebuild dependent packages automatically, it would still be the maintainer's responsibility to verify the results.